pyproject.toml

```diff
@@ -0,0 +1,97 @@
+[tool.black]
+line-length=119
+target-version=["py38"]
```

Comment:
Are you confident that the black formatting for Python 3.8 won't use formatting not compatible with Python 2.6?

Reply:
Tested with py26 unit tests and centos6 vagrant box